executor/insert_common.go

```diff
@@ -947,13 +948,12 @@ func (e *InsertValues) collectRuntimeStatsEnabled() bool {
 func (e *InsertValues) batchCheckAndInsert(ctx context.Context, rows [][]types.Datum, addRecord func(ctx context.Context, row []types.Datum) error) error {
```

Comment:
Please take a look at `InsertExec.exec` and `ReplaceExec.exec`, and recover the `insert into`, `replace`, `insert into duplicate` scenarios.